### PR TITLE
Porting Workflow Designer articles

### PR DESCRIPTION
**Pull request by justinchoi001**
cc @rbohl

### CODE REVIEW

File: docs/dxp/7.x/en/process-automation/workflow/user-guide/introduction-to-the-workflow-designer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Introduction to the Workflow Designer
+
+> Subscribers
+
+The Workflow Designer is an intuitive interface that lets you design workflow definitions so your assets go through a review process before publication. With the Workflow Designer, you develop workflow definitions using a convenient drag and drop user interface. You do not need to be familiar with writing XML definitions by hand. Some of the features can be enhanced with Groovy scripts, a supported Java-based scripting language.
+
+```tip::
+   By default, only one workflow definition is installed: the Single Approver Workflow definition.
```
Comment (rbohl) [inline docs/dxp/7.x/en/process-automation/workflow/user-guide/introduction-to-the-workflow-designer.md:8]: I don't think this is a tip. If I recall there's info from the original documentation on accessing the other workflows present in the kaleo runtime impl JAR. That would be a  tip, in my opinion.

Comment (justinchoi001) [reply docs/dxp/7.x/en/process-automation/workflow/user-guide/introduction-to-the-workflow-designer.md]: Russ, I added instructions on how to get to the other definitions.

**Comment by sez11a** [comment]
@justinchoi001 This is merge conflicted.

**Comment by justinchoi001** [comment]
@sez11a , Apologies. I resolved the conflicts and pushed up the changes.

**Comment by sez11a** [comment]
Just started reviewing :)

**Comment by sez11a** [comment]
Superseded by https://github.com/justinchoi001/liferay-learn/pull/27

**Pull request closed without merging**